relay_rpc/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![deny(clippy::indexing_slicing)]
```

Comment:
These can be unsafe generally and easy to forget. People can `#[allow(clippy::indexing_slicing)]` for each instance to bypass.

Reply:
this can also be set on the manifest so it's applied to the entire codebase like so:

```toml
[lints.clippy]
indexing_slicing = "forbid"
```